### PR TITLE
Add cache control headers to PDC HTTP response

### PR DESCRIPTION
**Pull request by ycheng-aa**
JIRA: PDC-1577

### CODE REVIEW

**Comment by bliuredhat** [comment]
:+1:

File: pdc/settings.py [hunk 1]
```diff
@@ -118,11 +122,14 @@
     'pdc.apps.usage.middleware.UsageMiddleware',
     'pdc.apps.changeset.middleware.ChangesetMiddleware',
     'pdc.apps.utils.middleware.MessagingMiddleware',
-    'pdc.apps.utils.middleware.RestrictAdminMiddleware'
+    'pdc.apps.utils.middleware.RestrictAdminMiddleware',
+    'django.middleware.common.CommonMiddleware',
```
Comment (simozhan) [inline pdc/settings.py]: CommonMiddleware is already on line 113

**Comment by simozhan** [comment]
i don't have much experience with cache and i am following this guide https://docs.djangoproject.com/en/1.9/topics/cache/
It seems in this guide, a CACHES setting need to be added which i haven't seen. But your test runs ok. So is the CACHES necessary?

**Comment by ycheng-aa** [comment]
I added CACHES item but removed it. The CACHES could let Django use it's owe cache instead of calculate again. This commit looks only include generating cache control header. And I am also afraid if use Django CACHES, client may be no way to get the newest data. I am not sure. I hope the first commit of cache could make less impact.

File: pdc/settings.py (hunk 1 above)
Comment (ycheng-aa) [inline pdc/settings.py]: Fixed in 13acb9cf5b4a4d3. Thanks for the careful review.

**Comment by ycheng-aa** [comment]
And I will also abandon internal code part change. Because no need to generate cache table if don't set `CACHES` here.

**Comment by simozhan** [comment]
looks good to me then.

**Pull request merged**